src/box/lua/schema.lua

```diff
@@ -3677,30 +3677,101 @@ local function role_check_grant_revoke_of_sys_priv(priv)
     end
 end
 
-box.schema.role.grant = function(user_name, ...)
+local function box_schema_role_grant(user_name, ...)
     local uid = role_resolve(user_name)
     if uid == nil then
         box.error(box.error.NO_SUCH_ROLE, user_name)
     end
     role_check_grant_revoke_of_sys_priv(...)
     return grant(uid, user_name, ...)
 end
-box.schema.role.revoke = function(user_name, ...)
+
+local function box_schema_role_revoke(user_name, ...)
     local uid = role_resolve(user_name)
     if uid == nil then
         box.error(box.error.NO_SUCH_ROLE, user_name)
     end
     role_check_grant_revoke_of_sys_priv(...)
     return revoke(uid, user_name, ...)
 end
-box.schema.role.info = function(role_name)
+
+local function box_schema_role_info(role_name)
```

Comment:
It should be `box_schema_role_info_do` to make it clear that it is a helper function now. The same is for all functions above.

Reply:
Renamed the non-transactional functions.